### PR TITLE
Add adopter logo

### PR DESCRIPTION
**Pull request by zhiyingfang2022**
Signed-off-by: zhiying <zhiying.fang@daocloud.io>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ KubeEdge is used by many companies. If you are using KubeEdge in your organizati
 <a href="http://xh-iot.net/" border="0" target="_blank"><img alt="XH-iot" src="docs/adopters/xh-iot.png" height="70"></a> &nbsp; &nbsp; &nbsp;
 <a href="http://www.jylink.com/" border="0" target="_blank"><img alt="jylink" src="docs/adopters/jylink.png" height="70"></a> &nbsp; &nbsp; &nbsp;<br/><br/>
 <a href="http://www.ictnj.ac.cn/" border="0" target="_blank"><img alt="ictnj" src="docs/adopters/ictnj.svg " height="70"></a>&nbsp; &nbsp; &nbsp;<br/><br/>
+<a href="https://www.daocloud.io/" border="0" target="_blank"><img alt="daocloud" src="docs/adopters/daocloud.svg " height="70"></a>&nbsp; &nbsp; &nbsp;<br/><br/>
```
Comment (vincentgoat) [inline ADOPTERS.md:12]: Need to remove extra line breaks.
image

Comment (zhiyingfang2022) [reply ADOPTERS.md]: done

**Review by vincentgoat — APPROVED** [review]
/lgtm
cc @fisherxu

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**